src/Views/SimpleLoginView.php

```diff
@@ -279,4 +286,20 @@ public function setForgotYourPasswordLabel($forgotYourPasswordLabel)
     {
         $this->forgotYourPasswordLabel = $forgotYourPasswordLabel;
     }
+
+    /**
+     * @return ValueInterface|string
+     */
+    public function getTitleLabel()
+    {
+        return $this->titleLabel;
```

Comment:
Here, you should put instead:

```
return ValueUtils::val($this->titleLabel);
```

Otherwise, if you pass a ValueInterface object to the setter, it will not be evaluated to a string.

Consequently, the @return annotation returns only a string.